trace_decoder/src/core.rs

```diff
@@ -533,10 +547,147 @@ fn middle<StateTrieT: StateTrie + Clone>(
     Ok(out)
 }
 
+#[allow(unused_variables)]
```

Comment:
only because of the hanging `ger_data` when not targeting cdk-erigon

Reply:
I think we could make this tighter, but I'd prefer a different approach

Reply:
Note that with the latest suggestion on your `discuss` PR, the lint isn't triggered anymore because both conditional blocks would be living in the same binary